sdk/iot/Azure.ResourceManager.IotFirmwareDefense/samples/Generated/Samples/Sample_FirmwareAnalysisWorkspaceResource.cs

```diff
@@ -195,7 +195,7 @@ public async Task Update_WorkspacesUpdateMinimumSetGen()
         public async Task GenerateUploadUri_WorkspacesGenerateUploadUrlMaximumSetGen()
         {
             // Generated from example definition: specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/stable/2024-01-10/examples/Workspaces_GenerateUploadUrl_MaximumSet_Gen.json
-            // this example is just showing the usage of "Workspaces_GenerateUploadUri" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Workspaces_GenerateUploadUrl" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `Url`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/stable/2024-01-10/workspaces.json#L315